src/components/Features.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * Featurescomponent
+ * This is the features and some information
+ */
+
+/* Imports -------------------------------------------------------------------*/
+
+import Inferno from 'inferno';
+import Component from 'inferno-component';
+
+/* Local variables -----------------------------------------------------------*/
+
+const version = '^3.0.0';
+
+/* Component -----------------------------------------------------------------*/
+
+class VersionComponent extends Component {
+	constructor(props) {
+		super(props);
+	}
+
+	showVersion() {
+		alert(`The version is: ${ version }!`);
+	}
+
+	render() {
+		return (
+			<div class="container features">
+			<div class="row">
+				<div class="col-lg-6">
+					<img class="rounded-circle feature-img" src="./image/optimize.png" alt="Generic placeholder image">
```

Comment:
The linter doesn't seem to appreciate non-closed image tags. you need to add a `/` at the end- aka a self-closing tag.
ex: `<img src="test.jpg" />`